cmd/virt-launcher/virt-launcher.go

```diff
@@ -348,6 +348,8 @@ func main() {
 	qemuAgentUserInterval := pflag.Duration("qemu-agent-user-interval", 10, "Interval in seconds between consecutive qemu agent calls for user command")
 	qemuAgentVersionInterval := pflag.Duration("qemu-agent-version-interval", 300, "Interval in seconds between consecutive qemu agent calls for version command")
 	qemuAgentFSFreezeStatusInterval := pflag.Duration("qemu-fsfreeze-status-interval", 5, "Interval in seconds between consecutive qemu agent calls for fsfreeze status command")
+	keepAfterQemu := pflag.Bool("keep-after-qemu", false, "virt-launcher will be kept alive after qemu exits for debugging if set to true")
```

Comment:
does the user know or care that the pod exists when qemu dies? Does it make sense to say `virt-launcher will be kept alive after failure for debugging if set to true`? or is that too vague?

Reply:
Nice. Will take your suggestion.

Reply:
I have adjust the name of the command line option, the annotation key, and corresponding internal variable to be inline with 'keep after failure'.